test/format/format_config.c

```diff
@@ -1449,6 +1451,88 @@ config_statistics(void)
     }
 }
 
+/*
+ * config_chunk_cache --
+ *     Chunk cache configuration.
+ */
+static void
+config_chunk_cache(void)
+{
+    char buf[128], chunkcache_storage_path[512];
+    const char *chunkcache_type;
+
+    chunkcache_type = NULL;
+
+    /* Chunkcache does not work unless tiered storage is configured. */
+    if (!g.tiered_storage_config) {
+        if (config_explicit(NULL, "chunk_cache") && GV(CHUNK_CACHE))
+            testutil_die(EINVAL,
+              "%s: chunkcache cannot be enabled unless tiered storage is configured.", progname);
+        return;
+    }
+
+    if (!config_explicit(NULL, "chunk_cache")) {
```

Comment:
Shouldn't we also check so chunk_cache is only configured if tiered storage is also configured.  Or is there some other logic at play?

Reply:
If tiered storage is not configured then we return early from the function.
```
    /* Chunkcache does not work unless tiered storage is configured. */
    if (!g.tiered_storage_config) {
        if (config_explicit(NULL, "chunk_cache") && GV(CHUNK_CACHE))
            testutil_die(EINVAL,
              "%s: chunkcache cannot be enabled unless tiered storage is configured.", progname);
        return;
    }
```

Reply:
For cases where tiered storage is not configured - 
- If chunkcache is explicitly configured, we produce an error. 
- When relying on chunkcache to be implicitly set by the format, we exit early.